core/src/scss/utilities/functions/_color.scss

```diff
@@ -0,0 +1,53 @@
+@charset "UTF-8";
+
+//
+// color($color, $colormap: $core-colors, $theme: default)
+//
+// Return a color from a color map. If only a color is specified, then it returns a color from $core-colors.
+// If a color and a color map are specified, then it returns a color from the default theme of the color map.
+// If all 3 are specified, it returns a color from the specified theme of the color map.
+//
+// $color - String - Color variable name from a color map, e.g., brand, link--hover
+// $colormap - Color palette in the form of a SASS map, e.g., $main-nav-colors, $alert-colors. Default is $core-colors.
+// $theme - String - Name of the theme within a color map, e.g., dark, light. Default is default.
+//
+// Style guide: Functions.Color
+//
+
+@function color($color, $colormap: $core-colors, $theme: default) {
```

Comment:
I think the third parameter (theme) could be removed in favor of simply passing in a different colormap.

Reply:
Also, this is a great interface for color changes. YES THANK YOU!